src/app/shared/services/entry-form.service.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  Observable,
+  of
+} from 'rxjs';
+
+import {
+  concatMap,
+  delay
+} from 'rxjs/operators';
+
+import {
+  GiftFormData
+} from '../interfaces/gift-form-data';
+
+import {
+  GiftSaveData
+} from '../interfaces/gift-save-data';
+
+@Injectable()
+export class EntryFormService {
+  constructor() {
+  }
+
+  // Returns true after 3 seconds
+  public saveRecord(saveData: GiftSaveData, formData: GiftFormData): Observable<boolean> {
+    return of(true).pipe(
+      concatMap(item => of(item).pipe(
+        delay(3000)
+      ))
+    );
+  }
+
+  // Returns true after 3 seconds
+  public logCancel(): Observable<boolean> {
+    return of(true).pipe(
+      concatMap(item => of(item).pipe(
+        delay(3000)
```

Comment:
Maybe we can shorten these delays down a bit, maybe to 1 second?  It looks like for the cancel we may need to show the wait indicator from the gift entry form code.  I can work on doing that separately.

Reply:
Making this change now

Reply:
Made change down to 1 second of delay